modules/canvaskit/compile.sh

```diff
@@ -56,6 +49,18 @@ else
   BUILD_DIR=${BUILD_DIR:="out/canvaskit_wasm"}
 fi
 
+# Tracing will be disabled unless this flag is seen.
+if [[ $@ != *force_tracing* ]] ; then
```

Comment:
I would reverse the 'if' and 'else' clauses so the test is == instead

Reply:
Just copied from previous but let me see.